roles/deploy-aws-infra/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Maybe just add these to the `defaults` file instead (together with all the other variables as mentioned in comments above)

Reply:
Sure